projects/golang/go/build/conformanceTests/builderBaseRebuildBuildspec.yaml

```diff
@@ -0,0 +1,22 @@
+version: 0.2
+
+env:
+  variables:
+    ARTIFACTS_BUCKET: "eks-d-postsubmit-artifacts"
+    BUILDKIT_PLATFORMS: "linux/amd64"
+    GO_SOURCE_VERSION: "1.16"
+    IMAGE_NAME: "builder-base-development"
+    IMPORT_CACHE_REPOS: ""
+    UPDATE_BASE_IMAGE: "false"
+
+phases:
+  build:
+    commands:
+    - export GOLANG_RPM_DOWNLOAD_TARGETS=$(yq  e "to_entries | .[] | select(.key == \"*GOLANG_VERSION*\") | select(.value != \"$GO_SOURCE_VERSION*\") | .value |= \"download-golang-\" + . | .value" $CODEBUILD_SRC_DIR/builder-base/versions.yaml)
```

Comment:
this over-rides the golang versions we download to NOT download the one we're building in the previous pipeline step; that way, we skip the download, and just copy the RPMs over from the previous step (lines 19 and 20) prior to building the images